Presentation.Web/app/models/ViewModel/it-system-usage/system-usage-view-model.ts

```diff
@@ -1,11 +1,50 @@
 ﻿module Kitos.Models.ViewModel.ItSystemUsage {
+    import Select2OptionViewModel = ViewModel.Generic.Select2OptionViewModel;
 
     export interface ISensitiveDataLevelModel {
         value: number;
         textValue: string;
         text: string;
     }
 
+    export enum DataOption {
+        NO = 0,
+        YES = 1,
+        DONTKNOW = 2,
+        UNDECIDED = 3
+    }
+
+    export enum RiskLevel {
+        LOW = 0,
+        MIDDLE = 1,
+        HIGH = 2,
+        UNDECIDED = 3
+    }
+
+    export class DataOptions {
+        options: Select2OptionViewModel[];
+        constructor() {
+            this.options = [
+                <Select2OptionViewModel>{ id: 3, text: "&nbsp" },
```

Comment:
Hm, det er nok ok med tilføjelsen af blank = "3", men kønt ser det ikke ud

Reply:
Ja det er for at værdierne passer med vores database model og at det så også passer på frontend og giver logisk mening der.

Reply:
Jeg er enig I at vi ikke behøver gøre yderligere - et alternativ kunne dog have været at mappe værdierne i backend til en egen frontend DTO enum type med mere logiske værdier (blank ="0")